pyproject.toml

```diff
@@ -8,7 +8,7 @@ dynamic = ["version"]
 description = "LLM framework to build customizable, production-ready LLM applications. Connect components (models, vector DBs, file converters) to pipelines or agents that can interact with your data."
 readme = "README.md"
 license = "Apache-2.0"
-requires-python = ">=3.8,<3.13"
+requires-python = ">=3.8"
```

Comment:
Would it be best practice to add `,<3.14`?

Reply:
In hindsight, perhaps it was a mistake to restrict to Python <3.13 (done by in at https://github.com/deepset-ai/haystack/pull/8547). I would prefer not to officially support python 3.14 but leave users free to experiment and report errors when this version becomes available.

Reply:
sounds good to me!